lib/counter_culture/reconciler.rb

```diff
@@ -299,8 +299,8 @@ def join_clauses(where)
 
           # adds 'type' condition to JOIN clause if the current model is a
           # child in a Single Table Inheritance
-          if reflect.active_record.column_names.include?('type') &&
-              !model.descends_from_active_record?
+          if model.column_names.include?('type') &&
```

Comment:
I think it should be using `model.inheritance_column` instead of `'type'`, but I've never seen anyone using any other inheritance column aside from `type`, so probably not a real problem, though I could also fix it in this PR.